src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -132,4 +132,6 @@ CanonicalMostRead.propTypes = {
   endpoint: string.isRequired,
 };
 
+CanonicalMostRead.defaultProps = {};
```

Comment:
You can omit this line since there are no defaults.

Reply:
thank you ☺️